flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingMetricCollector.java

```diff
@@ -104,17 +106,18 @@ public CollectedMetricHistory updateMetrics(
 
         var jobDetailsInfo =
                 getJobDetailsInfo(ctx, conf.get(AutoScalerOptions.FLINK_CLIENT_TIMEOUT));
-        var jobUpdateTs = getJobUpdateTs(jobDetailsInfo);
+        var jobSwitchToRunningTs = getJobSwitchToRunningTs(jobDetailsInfo);
```

Comment:
Can we keep the old name `jobUpdateTs`? Or rename to `jobRunningTs`? This change is a bit unrelated to the original issue in this PR.

Reply:
> Can we keep the old name jobUpdateTs? Or rename to jobRunningTs? 

Renaming it to `jobRunningTs`.

> This change is a bit unrelated to the original issue in this PR.

That's why I split this PR into 2 commits. 

- The first commit is refactor, it gets the jobRunningTs from JobStatus.RUNNING instead of max timestamp.
- The second commit is core change of the original issue.